src/Service.Host/client/src/actions/purchaseOrdersApplicationStateActions.js

```diff
@@ -0,0 +1,13 @@
+import { StateApiActions } from '@linn-it/linn-form-components-library';
+import { purchaseOrdersActionTypes as actionTypes } from './index';
+import * as itemTypes from '../itemTypes';
+import config from '../config';
+
+export default new StateApiActions(
```

Comment:
Not 100% sure, but I don't think you need this? I think the purchaseOrdersActions will have a .fetchState() action already

Reply:
Oh yeah so it does, looks like we've done it for reqs and signing limits too. I didn't spot that on FetchApiActions before